ssz/sedes/serializable.py

```diff
@@ -1,5 +1,5 @@
 import abc
-import collections
+import collections.abc
```

Comment:
We are doing this to avoid naming conflicts with `Sequence` that we import from `typing`.